device_doctor/lib/src/health.dart

```diff
@@ -17,10 +17,11 @@ import 'utils.dart';
 Future<HealthCheckResult> closeIosDialog(
     {ProcessManager pm = const LocalProcessManager(),
     String deviceId,
-    platform.Platform pl = const platform.LocalPlatform()}) async {
-  Directory dialogDir = dir(path.dirname(Platform.script.path), 'tool', 'infra-dialog');
+    platform.Platform pl = const platform.LocalPlatform(),
+    String infraDialog = 'infra-dialog'}) async {
```

Comment:
nit: if you add a comma after `infra-dialog',` it will format the args to each be on their own line

Reply:
reformatted. Thanks.